UIImageColorsPlayground.playground/Sources/UIImageColors.swift

```diff
@@ -25,21 +25,33 @@ class PCCountedColor {
     }
 }
 
-extension UIColor {
+extension UIColor
+{
+    var components:[CGFloat]
```

Comment:
This seems redundant since ```color.cgColor``` component property is fixed and that is what we are using. What's the point of adding an extension to ```UIColor```?